nil/services/rpc/rawapi/server.go

```diff
@@ -37,6 +37,9 @@ type NetworkTransportProtocolRo interface {
 	GetNumShards() pb.Uint64Response
 
 	ClientVersion() pb.StringResponse
+
+	GetTxpoolStatus() pb.Uint64Response
+	GetTxpoolContent() pb.Uint64Response
```

Comment:
I don't understand what is going on here
In `LocalShardApi` the return type is `[]*types.TxnWithHash`, and here it is `pb.Uint64Response`. It looks like an obvious inconsistency (besides I don't see converters added for `[]*types.TxnWithHas`).